crates/notion-core/src/shell/mod.rs

```diff
@@ -0,0 +1,112 @@
+use std::fs::File;
+use std::io::Write;
+use std::path::Path;
+use std::str::FromStr;
+
+use semver::Version;
+
+use notion_fail::{NotionError, NotionFail, Fallible, ResultExt};
+
+use env;
+
+mod bash;
+
+pub(crate) use self::bash::Bash;
+
+pub enum Postscript {
+    Path(String),
+    ToolVersion { tool: String, version: Version }
+}
+
+/// Thrown when the shell name specified in the Notion environment is not supported.
```

Comment:
Copy/paste from UnrecognizedShellError?

Maybe "Thrown when Postscript file is not found"?

Reply:
Good catch!